doc/source/ray-air/examples/analyze_tuning_results.ipynb

```diff
@@ -0,0 +1,918 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(train_dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d996bdd",
+   "metadata": {},
+   "source": [
+    "Ray AIR allows us to define and tune preprocessors. For instance, we may want to decide if we should scale the \"Wilderness area\" parameters or the \"Soil type\" parameters. For both, we'll just try standard scalers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "ef7b2992",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Let's tune the preprocessors\n",
+    "prep_v1 = StandardScaler(columns=[f\"Wilderness_Area_{i}\" for i in range(4)])\n",
+    "prep_v2 = StandardScaler(columns=[f\"Soil_Type_{i}\" for i in range(40)])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "282a6078",
+   "metadata": {},
+   "source": [
+    "Since we'll be training a multiclass prediction model, we have to pass some information to XGBoost. For instance, XGBoost expects us to provide the number of classes, and multiclass-enabled evaluation metrices.\n",
+    "\n",
+    "With these parameters in place, we'll create a Ray AIR `XGBoostTrainer`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "bbece53a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# XGBoost specific params\n",
+    "params = {\n",
+    "    \"tree_method\": \"approx\",\n",
+    "    \"objective\": \"multi:softmax\",\n",
+    "    \"eval_metric\": [\"mlogloss\", \"merror\"],\n",
+    "    \"num_class\": 8,\n",
+    "}\n",
+    "\n",
+    "trainer = XGBoostTrainer(\n",
+    "    scaling_config={\"num_workers\": 2},\n",
+    "    label_column=\"target\",\n",
+    "    params=params,\n",
+    "    datasets={\"train\": train_dataset},\n",
+    "    num_boost_round=10,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e436035b",
+   "metadata": {},
+   "source": [
+    "The trainer contains a few default parameters. We can now create the Tuner with a search space for other parameters. During the run, it will then overwrite the default parameters in the trainer according to the search specification.\n",
+    "\n",
+    "Here, we just want to tune the XGBoost `max_depth` parameter. Additionally, we want to compare our two `preprocessor` pipelines. Lastly, we configure Tune to minimize the `train-mlogloss` metric. In random search, this doesn't affect the evaluated configurations, but it will affect our default results fetching for analysis later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "c5d00628",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tuner = Tuner(\n",
+    "    trainer,\n",
+    "    param_space={\n",
+    "        \"params\": {\"max_depth\": tune.randint(2, 8)},\n",
+    "        \"preprocessor\": tune.grid_search([prep_v1, prep_v2]),\n",
+    "    },\n",
+    "    tune_config=TuneConfig(num_samples=4, metric=\"train-mlogloss\", mode=\"min\"),\n",
```

Comment:
I am wondering if it makes sense to echo back to the code above and point out why the metric is `train-mlogloss`

Reply:
Also do we want to briefly mention that Tune's optimization is based on single metric although xgboost reports two to Tune?

Reply:
Good points, added